src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -21,15 +21,27 @@
 from cryptography.hazmat.primitives.serialization import load_pem_private_key, Encoding, PublicFormat
 from cryptography.exceptions import UnsupportedAlgorithm
 
-
 from azure.cli.core import telemetry
 from azure.cli.core.profiles import ResourceType
+from azure.cli.core.util import sdk_no_wait
 
 from ._validators import _construct_vnet, secret_text_encoding_values
 
 logger = get_logger(__name__)
 
 
+def _azure_stack_wrapper(cmd, client, function_name, *args, **kwargs):
+    no_wait = False
+    if 'no_wait' in kwargs:
+        no_wait = kwargs.pop('no_wait')
+
+    if cmd.cli_ctx.cloud.profile == 'latest':
+        function_name = 'begin_' + function_name
```

Comment:
this is also specific to track2 right?

Reply:
@yungezz Yes, track 2 only.

Reply:
it only applies to latest profile and for existing 2019-03-01-hybrid profile with api version 2016-10-01, you will still use track 1 SDK, right?
But there is a plan to create a new profile named 2020-09-01-hybrid with api version 2019-09-01, if new profile is used you will still go to track1 SDK, is it expected?

Reply:
@Juliehzl No, all profiles after this commit will use track 2. For the new plan we also use track 2. Track 1 would be abandoned at this point. (Management-plane only)